README.rst

```diff
@@ -130,16 +130,35 @@ the testsuite with the following command:
 
 .. code-block:: bash
 
-        python setup.py test
+    python setup.py test
 
 Otherwise using a virtualenv is recommended. Setuptools will automatically fetch
 missing test dependencies:
 
 .. code-block:: bash
 
-        virtualenv
-        source .venv/bin/active
-        (venv) python setup.py test
+    virtualenv
+    source .venv/bin/active
+    (venv) python setup.py test
+
+Development
+-----------
+
+* Make sure that you have Java_ installed.
```

Comment:
Great, I agree with this migration procedure. I use `py.test` everywhere in my projects